environments/helm-values/values-int.yaml

```diff
@@ -263,13 +263,10 @@ backend:
       universalResolverAddress: "https://dev.uniresolver.io/"
       encryptionConfigs:
         index0:
-          encryptionKey: "<path:portal/data/int/encryption-keys#dim-encryption-key0>"
+          encryptionKey: "<path:portal/data/int/processes-worker#dim-encryption-key0>"
```

Comment:
for the association env the path needs be `<path:portal/data/int/encryption-keys#dim-encryption-key0>`

Reply:
adjusted the paths in the environment files